config/config/sc-config.yaml

```
@@ -259,40 +260,86 @@ dex:
     # For more details, see https://elastisys.com/elastisys-engineering-how-to-use-dex-with-google-accounts-to-manage-access-in-kubernetes/
     # SASecretName: set-me
 
-kibana:
-  subdomain: kibana
-  # Note sso is enabled via `elasticsearch.sso.enabled`
-  resources:
-    requests:
-      cpu: 100m
-      memory: 286Mi
-  nodeSelector: {}
-  tolerations: []
-  affinity: {}
+opensearch:
+  # 'subdomain' is set in common-config.yaml.
 
-elasticsearch:
-  subdomain: elastic
-  snapshotRepository: elastic-snapshots
-  # Create initial indices upon first startup
+  clusterName: opensearch
+
+  # Create initial indices upon first startup.
   createIndices: true
-  # Single-sign-on using OIDC
-  # NOTE: SSO using OIDC requires LetsEncrypt Production
+
+  dashboards:
+    subdomain: opensearch
+    # Note sso is enabled via `opensearch.sso.enabled`.
+    resources:
+      requests:
+        memory: 286Mi
+        cpu: 100m
+      limits:
+        memory: 1024Mi
+        cpu: 1
+    affinity: {}
+    nodeSelector: {}
+    tolerations: []
+
+  # Single-sign-on using OIDC, note this requires LetsEncrypt Production.
   sso:
     enabled: false
-    # Where to find subject
-    subject_key: email
-    # Where to find roles
-    roles_key: groups
-    # Scope - add 'groups' if groups claim is supported
+    subject_key: email # Where to find subject.
+    roles_key: groups  # Where to find roles.
+    # Scope - add 'groups' if groups claim is supported.
     scope: openid profile email
 
+  extraRoles: []
+  # - role_name: log_reader
+  #   definition:
+  #     index_permissions:
+  #     - index_patterns:
+  #       - "kubernetes-*"
+  #       allowed_actions:
+  #       - "read"
+
+  extraRoleMappings:
+    - mapping_name: readall_and_monitor
+      definition:
+        users:
+          - set-me # Developer Name
+    - mapping_name: kibana_user
```

Maybe? Or have they kept the `kibana_user` in the defaults? 
```suggestion
    - mapping_name: dashboards_user
```

---

Yeah, `kibana_user` is still a static role, the same with `kibana_server`.